src/NUnitFramework/nunitlite.tests/nunitlite.tests.csproj

```diff
@@ -23,7 +25,17 @@
   </ItemGroup>
 
  <ItemGroup>
-    <PackageReference Include="NUnit.Analyzers" />
+    <PackageReference Include="NUnit.Analyzers">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
```

Comment:
This is not needed. The Analyzer package is marked as a `developmentDependency` which effectively [does the same](https://docs.microsoft.com/en-us/nuget/reference/nuspec#developmentdependency) without having to specify it.

Reply:
Awesome!